### PR TITLE
Search Landing and Results

### PR DESCRIPTION
**Pull request by alex-dinh**

### CODE REVIEW

File: frontend/src/pages/Search/Search.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ import SearchBox from '../../components/SearchComponents/SearchBox';
 import SortBy from '../../components/SortBy/SortBy';
 
 const querystring = require('querystring');
-const DEBUG = false;
+const DEBUG = true;
```
Comment (micheyale) [inline frontend/src/pages/Search/Search.js:12]:
```suggestion
const DEBUG = false;
```

File: frontend/src/components/Auth/SignIn.js [hunk 2]
```diff
@@ -1,10 +1,6 @@
 import React from 'react';
-// import { Form, Divider, Button } from 'semantic-ui-react';
-// import firebase, { auth } from '../../firebase';
 import { Form } from 'semantic-ui-react';
 import { auth } from '../../firebase';
-// import { uiConfig } from '../../firebase';
-// import StyledFirebaseAuth from 'react-firebaseui/StyledFirebaseAuth';
```
Comment (EvelynL09) [inline frontend/src/components/Auth/SignIn.js:7]: Can we leave these line here?

**Review by micheyale — APPROVED** [review]

File: frontend/src/semantic-ui/site/modules/tab.overrides [hunk 3]
```diff
@@ -1,3 +1,8 @@
 /*******************************
         User Overrides
 *******************************/
+.ui.tab.active {
+    border: none;
+    border-radius: 0;
+    background-color: #F1F2F5;
+}
```
Comment (EvelynL09) [inline frontend/src/semantic-ui/site/modules/tab.overrides:8]: Could you check if this will affect My Profile page?

Comment (micheyale) [reply frontend/src/semantic-ui/site/modules/tab.overrides:8]: If it does, I will adjust it

File: frontend/src/components/Auth/SignIn.js (hunk 2 above)
Comment (micheyale) [reply frontend/src/components/Auth/SignIn.js:7]: Let me bring this code back

Comment (micheyale) [reply frontend/src/components/Auth/SignIn.js:7]: I will do another pull request and have line 6-7 back

**Pull request merged**